### ISSUE TITLE
Hi, I'm running Manjaro 17.1 Xfce and brackets 1.9.4. I noticed that the editor was not very responsive, until I disabled "show whitespace" option.  Now it runs fast, but I really miss showing white space.

### ISSUE THREAD

**Issue by Manjaroxfce** [adobe/brackets#13389]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported?
* [ ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?

For more information on how to write a good bug report read [here](https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue)
For more information on how to contribute read [here](https://github.com/adobe/brackets/blob/master/CONTRIBUTING.md)

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### Versions

Please include the OS and what version of the OS you're running.
Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)

**Comment by petetnt** [comment]
Hi @Manjaroxfce,

I modified the title a bit. You are using the Show Whitespace-extension correct? It's a known issue that CodeMirror overlays can kill performance (as noted in https://github.com/DennisKehrig/brackets-show-whitespace#known-issues), because it literally creates thousands and thousands of new DOM-nodes when enabled.

**Comment by Manjaroxfce** [comment]
Yes, I have the extention installed. I'm using Sublime Text and Atom.  but I like the Live Preview.  White Space showing is not a problem in Sublime and Atom. Thanks for your time.
